docs/modules/k3s.md

```diff
@@ -59,7 +59,7 @@ The `GetKubeConfig` method returns the K3s cluster's `kubeconfig`, including the
 to the Kubernetes Rest Client API using a Kubernetes client. It'll be returned in the format of `[]bytes`.
 
 <!--codeinclude-->
-[Get KubeConifg](../../modules/k3s/k3s_test.go) inside_block:GetKubeConfig
+[Get KubeConifg](../../modules/k3s/k3s_example_test.go) inside_block:GetKubeConfig
```

Comment:
There was a typo even before your changes 🤦  Let's fix it here too:
```suggestion
[Get KubeConfig](../../modules/k3s/k3s_example_test.go) inside_block:GetKubeConfig
```

Reply:
I will merge it right after the change, not waiting for the build, as everything is fine ✅